src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String INDEX = " ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFrom1 = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateTo1 = LocalDate.parse(dateTo, formatter);
+        int salary = 0;
+        StringBuilder sb = new StringBuilder();
```

Comment:
shortening variables names is almost never beneficial, readability is priority number one in this matter